.gitlab-ci.yml

```diff
@@ -30,7 +30,7 @@ compile:
   extends:
     - .global-vars
   script:
-    - go build -ldflags "-extldflags '-static'" -o "${CI_PROJECT_DIR}/${RELEASE_BIN}" ./cmd/alsaRemoteControl
+    - go build -ldflags "-extldflags '-static'" -o "${CI_PROJECT_DIR}/${RELEASE_BIN}" .
```

Comment:
Actually, just noticed:
if you don't rename `cmd/alsaRemoteControl/main.go -> main.go`
then you can build with, for example:
`GOOS=linux GOARCH=arm GOARM=6 go build ./cmd/alsaRemoteControl`
which gives you a default binary file of `alsaRemoteControl`

So I kind of like that approach

Reply:
The binary name is set by `go.mod` actually. So running `go build .` prior to this PR results in a no-op believe it or not. Thats what I was trying to fix.

Reply:
Oh, hmm, I guess the name is the same for both, but I have seen the `cmd/<binName>` approach in several other projects and it seems nice.

Reply:
With this changes in place, running `go build .` in the project root results in the `alsaRemoteControl` binary being outputted to the project root by default.

Reply:
I think you need that for `go install`? but that might be dated info, I can't recall

Reply:
Yea, theres the go install factor, but I think its common practice to always have some main entry point in the root of your proj dir if your proj is directly executable?

Reply:
We could go back and fourth but TBH ive seen a main.go in the proj root more often then not. if you dont like it then we can revise the PR :)

Reply:
Gotcha, that's a fair point. This is fine as you have it then.